stories/2-Card.stories.js

```diff
@@ -130,3 +133,30 @@ export const NoteCard = () => (
     </FlexCenter>
   </Card>
 );
+
+export const ContextCard = () => (
+  <ContextStyle>
+    <Flex>
+      <div className="context-div">
+        <h1>What is onboarding?</h1>
+        <p>abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz</p>
+      </div>
+      <DoneChecking src={done_checking} />
+    </Flex>
+  </ContextStyle>
+);
+
+export const HintCard = () => (
+  <React.Fragment>
+    <HintStyle style={{minWidth: "15%", width: "15%"}}>
+      <div id="context-div" style={{padding: "0 0 20px"}}>
+        <h1>Hint</h1>
+        <p>abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz</p>
+      </div>
+      <DoneChecking style={{width: "80%"}} src={done_checking}/>
+    </HintStyle>
+    <HintStyle style={{border: "none", minWidth: "15%", width: "15%"}}>
+      <PrimaryWithText style={{minWidth: "15%", width: "15%"}}/>
+    </HintStyle>
```

Comment:
Why the inline styles?

Reply:
actually disregard. It is not worth addressing here, I can address myself in a future PR when I implement this.